Doc/library/importlib.resources.rst

```diff
@@ -97,7 +105,11 @@ The following functions are available.
     sub-resources (i.e. it cannot be a directory).  This function returns a
     ``typing.BinaryIO`` instance, a binary I/O stream open for reading.
 
-   .. deprecated:: 3.11
+    .. deprecated:: 3.11
```

Comment:
```suggestion
    .. deprecated-removed:: 3.11 3.13
```

Reply:
I was going from https://github.com/python/importlib_resources/issues/80, which says 3.13. I think it is better for users to have a concrete version for removal listed, even if a decision is taken later to delay the removal.

A

Reply:
IMO this should be a separate change, not part of cleaning up docs.